src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -324,6 +324,8 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
   <PropertyGroup>
     <GenerateBindingRedirectsOutputType Condition="'$(OutputType)'=='exe' or '$(OutputType)'=='winexe'">true</GenerateBindingRedirectsOutputType>
+    <!-- It would be a breaking change to automatically turn on binding redirects for existing projects, so turn them on only when opting into a new framework. -->
+    <AutoGenerateBindingRedirects Condition="'$(AutoGenerateBindingRedirects)' == '' and '$(TargetFrameworkIdentifier)' == '.NETFramework' and '$(TargetFrameworkVersion.Substring(1))' >= '4.7.2'">true</AutoGenerateBindingRedirects>
```

Comment:
Another reference to `TargetFrameworkVersion.Substring(1)` indicates that there's not always a leading "v".

https://github.com/Microsoft/msbuild/blob/master/src/Tasks/Microsoft.Common.CurrentVersion.targets#L1898-L1899

Does this need similar logic?

Reply:
I don't know why that logic is the way it is. All the other comparisons assume it starts with a `v`. But yeah, should be cautious.